src/axom/primal/geometry/Triangle.hpp

```diff
@@ -330,18 +331,29 @@ class Triangle
   }
 
   /*!
-   * \brief Returns whether Point P is in the triangle for some 3d Triangle
-   * \return true iff P is in the triangle
-   * \see primal::Point
+   * \brief Returns whether Point \a p is contained within the triangle
+   * 
+   * \return true if P is in the triangle
+   * \deprecated This function is deprecated. Use \a contains() instead
+   */
+  AXOM_PRIMAL_DEPRECATED bool checkInTriangle(const PointType& p, double eps = 1e-8) const
+  {
+    return isInside(p, eps);
```

Comment:
Should this call "contains"?

Reply:
Thanks -- that was leftover from a previous iteration